src/Button/Button.Component.js

```diff
@@ -86,6 +92,14 @@ const clickBtnHandler = btn => {
     Disabled State
 </Button>
 
+<Button type="medium">Normal State</Button>
```

Comment:
@leannerivera I think we forgot to remove the buttons we were playing with 😂

Reply:
Yeah, gonna do that now. xD